internal/printer/printer.go

```diff
@@ -512,6 +514,32 @@ func maybeConvertTransition(n *astro.Node) {
 			})
 		}
 	}
+	if transform.HasAttr(n, transform.TRANSITION_RELOAD) {
+		transitionReloadIndex := transform.AttrIndex(n, transform.TRANSITION_RELOAD)
+		attr := n.Attr[transitionReloadIndex]
```

Comment:
I noticed that `attr` isn't declared as a pointer. Since we're modifying the key in line 533, you might want to receive a pointer to ensure the changes take effect.

```suggestion
		attr := &n.Attr[transitionReloadIndex]
```

Perhas adding a test to ensure its presence like this one would be a good idea?https://github.com/withastro/compiler/blob/c17734d1607d83cd357a5b6204f33fd37ebe1782/internal/printer/printer_test.go#L3412-L3419

Reply:
oh dear, thank you very much for this hint, good catch!

Reply:
Pointers are fun :)

Reply:
@MoustaphaDev test added to printer_test.go

Reply:
Awesome, love the refactor to an early return pattern too 🤩!